core/src/test/java/hudson/FilePathTest.java

```diff
@@ -368,7 +370,13 @@ private void checkTarUntarRoundTrip(String filePrefix, long fileSize) throws Exc
         // Decompress
         FilePath outDir = new FilePath(temp.newFolder(filePrefix + "_out"));
         final FilePath outFile = outDir.child(tempFile.getName());
-        tmpDirPath.child(tarFile.getName()).untar(outDir, TarCompression.NONE);
+        final Logger LOGGER = Logger.getLogger("testing");
+
+        Consumer<List<String>> function = (list) -> {
+            list.stream().forEach(f ->LOGGER.info("Unstashing "+ f));
```

Comment:
would be better to have a real test on this rather than printing log.
maybe some assert on the file entries.

Reply:
also `archive` is used by tar which is widely used in the tests.